pkg/kubelet/BUILD

```
@@ -25,6 +25,7 @@ go_library(
         "kubelet_resources.go",
         "kubelet_volumes.go",
         "pod_container_deletor.go",
+        "pod_sandbox_deleter.go",
```

nit: in order to match the previous name:

```suggestion
        "pod_sandbox_deletor.go",
```